### PR TITLE
Completed html-forms exercise. -Chris Mendoza

### PR DESCRIPTION
**Pull request by CTMendoza**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<html>
+
+<head>
+  <title>HTML-Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    <form action="index.html">
```
Comment (BrettAlbright) [inline html-forms/index.html]: There should just be one `form` element for the whole page

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      <div><label for="Email">Email</label></div>
+      <input type="text" name="Email" id="Email">
+    </form>
+  </div>
+  <div>
+    <form action="index.html">
+      <div>Why is coding cool?</div>
```
Comment (BrettAlbright) [inline html-forms/index.html]: missing label here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
close! Just a few updates!

File: html-forms/index.html (hunk 2 above)
Comment (CTMendoza) [reply html-forms/index.html]: html-forms2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**